FiveCalls/FiveCallsTests/ContactLogsTests.swift

```diff
@@ -31,7 +31,7 @@ class ContactLogsTests: XCTestCase {
         let dateFormatter = DateFormatter()
         dateFormatter.dateFormat = "yyyy-MM-dd"
         let date = dateFormatter.date(from: "1984-01-24")!
-        let log = ContactLog(issueId: "issue1", contactId: "contact1", phone: "111-222-3333", outcome: .voicemail, date: date)
+        let log = ContactLog(issueId: "issue1", contactId: "contact1", phone: "111-222-3333", outcome: R.string.localizable.outcomesVoicemail(), date: date)
```

Comment:
This should not be localized, it's a key sent from the server. Can replace here with 'voicemail'.

Reply:
Or maybe I don't understand how this is generated in R.swift? I'm not clear what this will actually return.